ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -5734,3 +5734,40 @@ def test_torch_special_and(
         frontend=frontend,
         on_device=on_device,
     )
+
+    # bitwise_xor
+
+
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_xor",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
```

Comment:
We should test the method with Boolean inputs as well

Reply:
Yep, I have added that in new commit